src/components/Header.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable */
+
```

Comment:
- [ ]  For best practice, kindly make sure you fix your linters errors instead of dasabling them for the entire file.

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)

Reply:
@tresorsawasawa Thanks for the review. I know how important eslint is, but I disabled it because the linter errors only appeared on GitHub and not on my vscode. I didn't know how else to fix it on GitHub since I had already fixed it in my local repo.(That is, my app ran locally without errors of any sort)